src/tests/suite/daos_pool.c

```diff
@@ -491,7 +491,7 @@ pool_properties(void **state)
 	rc = daos_pool_query(arg->pool.poh, NULL, NULL, prop_query, NULL);
 	assert_int_equal(rc, 0);
 
-	assert_int_equal(prop_query->dpp_nr, 5);
+	assert_int_equal(prop_query->dpp_nr, 7);
```

Comment:
I know these magic numbers were already there and you were incrementing them but is there a define we could use instead somewhere?

Reply:
I can move the define that's in pool/srv_layout.c.